Cargo.lock

```diff
@@ -563,13 +562,15 @@ dependencies = [
 
 [[package]]
 name = "crates-io"
-version = "0.36.1"
+version = "0.37.0"
 dependencies = [
- "anyhow",
  "curl",
+ "http-auth",
+ "log",
  "percent-encoding",
  "serde",
  "serde_json",
+ "thiserror",
```

Comment:
`thiserror` is already in the dependency graph. Just do we want it to appear as a direct dependency or not.